### PR TITLE
feat/dataloader and simple training script

### PR DESCRIPTION
**Pull request by mlederbauer**
This is a simple core structure code for training, plotting and analyzing a model.

`scripts/train_simple_RF.py`
- load dataset from huggingface
- transform file_name column into columns with ligand names (this needs to be processed further for data processing, i.e. one-hot encoding the ligand features)
- split and scale data
- RandomForestRegressor pipeline
- R2 score and simple predicted_vs_ground_truth plot

`nmrcraft/utils/set_seed.py`
- set numpy seed (maybe torch later if we use it)

`nmrcraft/analysis/plotting.py`
- load matplotlib style
- simple predicted vs ground truth plot

`nmrcraft/data/dataset.py`
- load data from HF repo
- get ligand columns (maybe needs to be one-hot encoded later for classification problems)
- split data

I think this is a good place to start with building the model pipeline, most work needs to be done in the data loader, evaluation, plotting.

### CODE REVIEW

**Review by tiaguinho-code — APPROVED** [review]
Looks good from my side 👍🏼😎
